### DISCUSSION TITLE
Investment Timeline Calculations.

### DISCUSSION THREAD

**Discussion by soloam** [ghostfolio/ghostfolio#3860] [Q&A]
Regarding the Investment Timeline graph, what is the math on those values, for example:

![image](https://github.com/user-attachments/assets/4add20ab-778e-4937-90c9-88cb6043c759)
 
These are all the buy to October so far, and If I get the graph of the investments, it will show:

![image](https://github.com/user-attachments/assets/3f4a185d-c582-42ce-a60d-bfbaa3b736cd)

The values don't add up, it shows 160,67 Euro! In EURO it should show around 199 Euros.

What I'm missing?

Tks

**Comment by dtslvr** [comment]
Have you sold anything in the same period?

**Reply by soloam** [reply]
No, those are the only movements that I have on October!

**Reply by dtslvr** [reply]
Ok. Could you please create a bug report and attach a minimal export file to reproduce this scenario?

**Comment by soloam** [comment]
This was a problem with the time zone, my investments and activities where off 1 hour because of wrong time zone! I added this to my docker compose env file:

`
TZ=Europe/Lisbon
`

Tks
